controllers/suite_test.go

```diff
@@ -112,9 +112,6 @@ var _ = BeforeSuite(func() {
 		Fail("Environment variable TFC_TOKEN is required, but either not set or empty")
 	}
 	// Terraform Cloud Client
-	tfClient, err = tfc.NewClient(&tfc.Config{Token: os.Getenv("TFC_TOKEN")})
-	Expect(err).ToNot(HaveOccurred())
-	Expect(tfClient).ToNot(BeNil())
```

Comment:
LGTM given that the tests pass. Looks like [this code here](https://github.com/hashicorp/terraform-cloud-operator/blob/2d53ae3653398ecadd2e76ae1b54bde73b46752f/controllers/suite_test.go#L132-L134) is what really matters.